doc/dataset/modules/ROOT/pages/changing-provider.adoc

```diff
@@ -18,7 +18,18 @@ The recommended setup is to first complete the guide xref:installation/installat
 
 Developing the provider against Keycloak Wildfly, or with a locally running instance of Keycloak Quarkus is possible as well, still it is not part of this guide today.
 
-== Preparing changes to the provider
+== Preparing changes using the Minikube setup
```

Comment:
Should this be `Preparing changes using the local setup` of sorts ? @ahus1

Reply:
Oops, a copy-and-paste error! I pushed a change to correct this in the way you suggested.